### ISSUE TITLE
Board Review: Azure Communication Services (SPOOL) Auth CTE (JS, Python, .NET, Java)

### ISSUE THREAD

**Issue by garchiro7** [Azure/azure-sdk#4272]
Responsible:
petr.svihlik@microsoft.com

Team:
[ACS - Auth]

Champion scenarios:

API View:

[[.Net] Preparing CTE for the GA by AikoBB · Pull Request #26924 · Azure/azure-sdk-for-net (github.com)](https://github.com/Azure/azure-sdk-for-net/pull/26924)
	
[[JS] Preparing CTE for the GA by AikoBB · Pull Request #21571 · Azure/azure-sdk-for-js (github.com)](https://github.com/Azure/azure-sdk-for-js/pull/21571)
	
[[Java] Preparing CTE for the GA by AikoBB · Pull Request #26924 · Azure/azure-sdk-for-java (github.com)](https://github.com/Azure/azure-sdk-for-java/pull/26924)
	
[[Python] Preparing CTE for the GA by AikoBB · Pull Request #22987 · Azure/azure-sdk-for-python (github.com)](https://github.com/Azure/azure-sdk-for-python/pull/22987)

**Comment by lilyjma** [comment]
scheduled for 5/12 9-11am

**Comment by petrsvihlik** [comment]
@mpodwysocki absolutely, the code was grabbed from my dirty PoC which I just started converting to async/await for better readability. we'll take your comment into consideration when polishing the final samples. thanks for pointing it out!

**Comment by tg-msft** [comment]
[Recording](https://msit.microsoftstream.com/video/97c90840-98dc-b654-eda2-f1ecd543906c) (MS INTERNAL ONLY)

**Comment by petrsvihlik** [comment]
all api views approved & PRs merged
